### PR TITLE
Converted lat and long grid output to degrees from radians

### PR DESCRIPTION
**Pull request by skandalaCLIMA**
Coverted lat and long grid output to degrees from radians. Shifted the lat range to -90 -> 90 from 0 -> 180.

### CODE REVIEW

File: src/Mesh/Interpolation.jl [hunk 1]
```diff
@@ -511,9 +511,9 @@ struct InterpolationCubedSphere{FT <: AbstractFloat,
         nblck = nhor * nhor * nvert
         Δh = 2 / nhor                               # horizontal grid spacing in unwarped grid
 
-        lat_grd  = range(lat_min,  lat_max,  step=lat_res)
-        long_grd = range(long_min, long_max, step=long_res)
-        rad_grd  = range(rad_min,  rad_max,  step=rad_res)
+        lat_grd  = collect(range(lat_min,  lat_max,  step=lat_res))
```
Comment (simonbyrne) [inline src/Mesh/Interpolation.jl]: lets use degrees (and the correct ranges) everywhere, i.e. set
```julia
lat_min,   lat_max = FT(-90), FT(90)
long_min, long_max = FT(-180), FT(180)
```
and use the "degree" trig functions (`sind`, `cosd`, etc)

File: src/Mesh/Interpolation.jl [hunk 2]
```diff
@@ -498,8 +498,8 @@ struct InterpolationCubedSphere{FT <: AbstractFloat,
         toler2 = FT(eps(FT) * 4.0)                 # tolerance 
         toler3 = FT(eps(FT) * vert_range[1] * 10.0) # tolerance for Newton-Raphson 
 
-        lat_min,   lat_max = FT(0.0), FT(π)                 # inclination/zeinth angle range
-        long_min, long_max = FT(0.0), FT(2*π)  			    # azimuthal angle range
+        lat_min,   lat_max = FT(-90.0), FT(90.0)                # inclination/zeinth angle range
+        long_min, long_max = FT(0.0), FT(360.0)  			    # azimuthal angle range
```
Comment (simonbyrne) [inline src/Mesh/Interpolation.jl]:
```suggestion
        long_min, long_max = FT(-180), FT(180)  			    # azimuthal angle range
```

File: src/Mesh/Interpolation.jl (hunk 1 above)
Comment (simonbyrne) [inline src/Mesh/Interpolation.jl]: `collect` should now be unnecessary.

File: src/Mesh/Interpolation.jl [hunk 3]
```diff
@@ -557,10 +557,13 @@ struct InterpolationCubedSphere{FT <: AbstractFloat,
             end
             #--------------------------------
             for j in 1:n_lat
-                @inbounds x3_grd = rad * cos(lat_grd[j])
+#                @inbounds x3_grd = rad * cos(lat_grd[j])
```
Comment (simonbyrne) [inline src/Mesh/Interpolation.jl]: get rid of comments

File: test/Mesh/interpolation.jl [hunk 4]
```diff
@@ -266,9 +266,9 @@ function run_cubed_sphere_interpolation_test()
         x1g = Array{FT}(undef,nrad,nlat,nlong); x2g = similar(x1g); x3g = similar(x1g)
 
         for k in 1:nlong, j in 1:nlat, i in 1:nrad
-            x1g[i,j,k] = rad[i] * sin(lat[j]) * cos(long[k]) # inclination -> latitude; azimuthal -> longitude.
-            x2g[i,j,k] = rad[i] * sin(lat[j]) * sin(long[k]) # inclination -> latitude; azimuthal -> longitude.
-            x3g[i,j,k] = rad[i] * cos(lat[j])
+            x1g[i,j,k] =  rad[i] * cosd(lat[j]) * cosd(long[k]) # inclination -> latitude; azimuthal -> longitude.
+            x2g[i,j,k] =  rad[i] * cosd(lat[j]) * sind(long[k]) # inclination -> latitude; azimuthal -> longitude.
+            x3g[i,j,k] = -rad[i] * sind(lat[j])
```
Comment (simonbyrne) [inline test/Mesh/interpolation.jl]: Why is this negated?

File: src/Mesh/Interpolation.jl (hunk 2 above)
Comment (skandalaCLIMA) [reply src/Mesh/Interpolation.jl]: Made recommended revisions.

File: src/Mesh/Interpolation.jl (hunk 3 above)
Comment (skandalaCLIMA) [reply src/Mesh/Interpolation.jl]: made recommended revisions.

File: test/Mesh/interpolation.jl (hunk 4 above)
Comment (skandalaCLIMA) [reply test/Mesh/interpolation.jl]: Inclination angle originally measured from z axis. Definition filled to use -pi/2 at south pole & pi/2 at North pole.

**Comment by simonbyrne** [comment]
I tweaked the minus signs, but my editor also cleaned up the trailing whitespace, hence the diff.

**Pull request merged**